contracts/two-party-pol-holder/src/msg.rs

```diff
@@ -0,0 +1,233 @@
+use std::fmt;
+
+use astroport::asset::Asset;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Api, Attribute, Coin, Decimal, StdError};
+use covenant_macros::{clocked, covenant_clock_address, covenant_next_contract};
+use covenant_utils::ExpiryConfig;
+
+use crate::error::ContractError;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub clock_address: String,
+    pub pool_address: String,
+    pub next_contract: String,
+    pub lockup_config: ExpiryConfig,
+    pub ragequit_config: RagequitConfig,
+    pub deposit_deadline: Option<ExpiryConfig>,
```

Comment:
Why is this an option? What happens if no deposit deadline is given?

Reply:
Good point, it doesn't need to be an option as the `ExpiryConfig` contains a `None` variant.
Regarding what happens if no deadline is given - nothing. It keeps on being available for deposits until they arrive or deposit deadline is updated to some value that will expire. Funds then get refunded.

Reply:
adressed in #125